pyomo/contrib/pyros/util.py

```diff
@@ -990,6 +995,13 @@ def output_logger(config, **kwargs):
                     "Please provide feedback and/or report any issues by opening a Pyomo ticket.\n"
                     "===========================================================================================\n")
     # === ALL LOGGER RETURN MESSAGES
+    if "bypass_global_separation" in kwargs:
+        if kwargs["bypass_global_separation"]:
```

Comment:
You could make this simpler with:
```python
if kwargs.get('bypass_global_separation', False):
    config.progress_logger.info( #...
```

(This could also be propagated to other tests in this method)